packages/core/core/src/Validation.js

```diff
@@ -71,7 +71,11 @@ export default class Validation {
     };
 
     let config = await this.loadConfig(configRequest);
-    let parcelConfig: ParcelConfig = nullthrows(config.result);
+    nullthrows(config.result);
```

Comment:
This was actually a regression bug by the packageManager...

Reply:
Flow should be able to better understand `invariant(config.result != null)` as it doesn't know that `nullthrows` throws, only that it returns something non-null.